.github/workflows/github.yml

```diff
@@ -63,6 +72,10 @@ jobs:
         working-directory: plugins/maven/example-client
         run: mvn package
 
+      - name: Build sbt client example
+        working-directory: plugins/sbt/graphql-java-codegen-sbt-plugin/sbt-codegen-exmaple
```

Comment:
Fix directory name: sbt-codegen-ex**ma**ple

Reply:
hava change to  equals with others plugins such as` plugins/sbt/example-client`

Reply:
There’s a typo: exmaple

Reply:
I  have change, but pr not change.I don't know what happened